src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Delete.cs

```diff
@@ -70,15 +67,14 @@ public void FileSystemWatcher_Directory_Delete_DeepDirectoryStructure()
         [ConditionalFact(typeof(MountHelper), nameof(MountHelper.CanCreateSymbolicLinks))]
         public void FileSystemWatcher_Directory_Delete_SymLink()
         {
-            using (var testDirectory = new TempDirectory(GetTestFilePath()))
-            using (var dir = new TempDirectory(Path.Combine(testDirectory.Path, "dir")))
-            using (var tempDir = new TempDirectory(GetTestFilePath()))
-            using (var watcher = new FileSystemWatcher(Path.GetFullPath(dir.Path), "*"))
+            string dir = CreateTestDirectory(TestDirectory, "dir");
+            string tempDir = CreateTestFile();
```

Comment:
```suggestion
            string tempDir = CreateTestDirectory(TestDirectory, "tempDir");
```

Reply:
have you tried out running tests in WSL2 -- so you can run tests on Linux locally? This will fail there (ought to fail on Windows too though)

Reply:
I haven't tried running them in WSL. They did not fail on Windows though

Reply:
@danmoseley instead of `CreateTestDirectory(TestDirectory, "tempDir")`, I just used `CreateTestDirectory()` since the original code suggests `TempDirectory(GetTestFilePath())`